build/actions.js

```diff
@@ -62,6 +63,7 @@ class Actions {
                 common: {
                     name: substate,
                     type: "boolean",
+                    def: false,
```

Comment:
this is required to have an initial state, which is required if you use a "toggle" instead of "set" URL